mockOSApi.v2/README.md

```diff
@@ -0,0 +1,7 @@
+To Do:
+
+- to finish all the entity objects
+- to implement mediator for communication between objects - process-threads etc
```

Comment:
I like that! That would be a good design pattern to use for this use case

Reply:
Yes, I need to think though if this will not add some useless overhead, just to prove I can use a meditator library lol